Weapons-Guide.md

```diff
@@ -9,17 +9,17 @@ In most games all players are equipped with a pistol and two of their favorite l
+| <img src="images/weapons/shotgun.png" width="128px"/> | **Shotgun** | The *super shotgun* shoots 25 (needs comfirming) pellets in a fairly wide spread to a fairly short effective range. For longer range damage, the secondary fire is capable of lobbing bleed-inducing shrapnel significently farther. While the primary is as deadly as anything at close range, the shotgun has to reload in pairs of bullets at a time, making its user vulnerable to attack if time is not afforded to allow the user to reload. |
+| <img src="images/weapons/smg.png" width="128px"/> | **Submachine Gun** | While hard to control, the *submachine gun* deals very heavy damage at short to medium range. At medium and longer ranges, the SMG can still do damage in bursts, but it is very difficult to maintain accuracy at long ranges at full auto. The SMG has an airbust secondary fire that hurls a ball of shrapnel in a medium distance. While a very useful medium range weapon, the submachine gun is prone to emptying its ammunition quickly if the user is careless, which can be exploited by attentive opponents. |
+| <img src="images/weapons/flamer.png" width="128px"/> | **Flamer** | Another ammunition-limited weapon, the *flamethrower* can deal immense damage very quickly via its stream of fire which lances out in front of it. The flamethrower is not a weapon for the careless though; the operator can quickly find themselves in the midst of their own weapon's flames if they are not prudent about avoiding them. The flamer also has a secondary attack, capable of launching opponents into the air; this is especially useful as a sword deterrent. |
+| <img src="images/weapons/plasma.png" width="128px"/> | **Plasma** | The *plasma inductor* has a medium range primary fire with a larger hit radius but a slow projectile speed. While it requires good shot-leading skills, its larger hit radius means it is capable of consistent, if unexceptional, damage rates. The secondary fire requires charging, but releases a large orb of plasma energy which pulls in cyborgs, both freindly and enemy alike and deals exceptionaly large amount of damage, usualy engough to kill. |
+| <img src="images/weapons/zapper.png" width="128px"/> | **Zapper** | The *zaper* is an extremely high firerate weapon with low reacoil and further range, but is lacking in damage. The secondary fire is a charged, tight spread, shotgun like blast that causes the shock |
+| <img src="images/weapons/rifle.png" width="128px"/> | **Rifle** | The longest ranged, highest damaging weapon in the game. When unscoped it deals a somewhat reasonable amount of damage but at a slow firerate. When scoped in has a long charge delay, uses 2 bullets, but deal substantialy more damage, enough to one-headshot an unbuffed opponent. However, it is heavy, has limited ammunition. |
```

Comment:
Same thing as the previous occurrence of `firerate`. `substantially` again. Maybe try `enough to immediately frag an unbuffed opponent with a headshot`. Might also want an `and` before `has limited ammunition`.